source/documentation/information/mojgithubenterprise.html.md.erb

```diff
@@ -1,47 +1,128 @@
+Services Organisations.
+
+If you require access to these organisations, you can contact us via our Slack
+channel [#ask-operations-engineering](https://mojdt.slack.com/archives/C01BUKJSZD4) or email us
+at [Operations Engineering](mailto:operations-engineering@digital.justice.gov.uk).
+
+> You must provide an email address that we can send the invitation to. We only allow access to the following
+pre-approved email addresses:
+>
+> - @cica.gov.uk
+> - @digital.justice.gov.uk
+> - @ima-citizensrights.org.uk
+> - @judicialappointments.gov.uk
+> - @judiciary.uk
+> - @justice.gov.uk
+> - @ppo.gov.uk
+> - @publicguardian.gov.uk
+> - @sentencingcouncil.gov.uk
+> - @yjb.gov.uk
+
+If you do not have a GitHub account, the invitation you receive will prompt you to create one before joining the
+organisation.
+
+If you already have a GitHub account and wish to use it for work, you should add your work email as
+your [primary email](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/changing-your-primary-email-address)
+address in GitHub.
+
+#### Single Sign On (SSO) for Ministry of Justice Organisation
+
+If you have a `@digital.justice.gov.uk` and only require access to the Ministry of Justice Organisation, you can use
+[SSO](https://github.com/orgs/ministryofjustice/sso) to gain access via your Google Workspace account.
+
+#### Third-Party Supplier Management
+
+If you are a Third-Party Supplier who is maintaining code on behalf of the Ministry of Justice, you will need to be
+added as
+an [Outside Collaborator](https://technical-guidance.service.justice.gov.uk/documentation/standards/storing-source-code.html#outside-collaborator).
+
+Outside Collaborators will only be able to gain access to specific repositories.
+
+If you require access/need to give access to an Outside Collaborator, raise an issue or pull request in the relevant
+repository for the organisation below:
+
+- Ministry of
+  Justice = [ministryofjustice/github-collaborators](https://github.com/ministryofjustice/github-collaborators)
+- MoJ Analytical
+  Services = [moj-analytical-services/github-outside-collaborators](https://github.com/moj-analytical-services/github-outside-collaborators)
+
+### Third-Party Application and Integration Approval
+We review and approve requests to integrate third-party apps, keeping our tech ecosystem secure and efficient.
 
-## Why We Use GitHub
+### Fine-Grained Personal Access Token Requests
 
-GitHub is a powerful tool for software development and collaboration. Here's why we use it:
+We oversee the approval of such requests, ensuring secure and proper usage of GitHub.
 
-- Version Control: GitHub is built on Git, which tracks changes to our code over time. This ensures that every revision is documented, and we can revert to an older version if necessary.
+## Safety Nets
```

Comment:
Perhaps explain what is meant by safety nets. This isn't clear.

Reply:
I agree! ✅ and I believe this page needs some additional investment into it 💸 

These categories were more for contextualising how I was going to lay out the joiners stuff, so I've just reverted the non-updated content back to the old structure for now 😁 

When someone can take a look at the other sections in a bit more detail, they can decide where it'll live 🏡